pkg/provisioner/ironic/ironic.go

```diff
@@ -1591,6 +1616,18 @@ func (p *ironicProvisioner) Provision(data provisioner.ProvisionData) (result pr
 		p.log.Info("finished provisioning")
 		return operationComplete()
 
+	case nodes.DeployWait:
+		if forceReboot {
+			p.log.Info("aborting provisioning to update the image")
```

Comment:
Maybe say "aborting provisioning to force reboot of preprovisioning image" since that's all we actually know for sure of the user's intentions.